code/modules/clothing/suits/bio.dm

```diff
@@ -69,20 +69,20 @@
 /obj/item/clothing/suit/bio_suit/scientist
 	icon_state = "bio_scientist"
 
-
 //CMO's biosuit, blue stripe
-/obj/item/clothing/suit/bio_suit/cmo
-	icon_state = "bio_cmo"
-
 /obj/item/clothing/head/bio_hood/cmo
 	icon_state = "bio_cmo"
 
+/obj/item/clothing/suit/bio_suit/cmo
+	icon_state = "bio_cmo"
+	allowed = list(/obj/item/tank/internals, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/reagent_containers/glass/beaker, /obj/item/melee/classic_baton/telescopic)
 
 //Plague Dr mask can be found in clothing/masks/gasmask.dm
 /obj/item/clothing/suit/bio_suit/plaguedoctorsuit
 	name = "plague doctor suit"
 	desc = "It protected doctors from the Black Death, back then. You bet your arse it's gonna help you against viruses."
 	icon_state = "plaguedoctor"
 	item_state = "bio_suit"
+	allowed = list(/obj/item/storage/book/bible, /obj/item/nullrod, /obj/item/cane)

```

Comment:
same as CMO comment, it should inher base on top of these tbh.

Reply:
not the cane though, idk why that's there.

Reply:
cane is here because plague doctors used canes to poke patients

it's not a 'real' biosuit so it shouldn't inherit anything imo, since it can be bought from costume vendors. if you feel strongly i'll change it

Reply:
it's suppose to be a biosuit hence the path.

Reply:
done